examples/generalized_linear_models/GLM-truncated-censored-regression.ipynb

```diff
@@ -0,0 +1,1356 @@
+{
```

Comment:
<p>Nitpick: why are you calling the slope <code>m</code> ? I found that a bit unintuitive</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>I guess it comes from the standard line equation: y = m*x + b (that's how I learned it, but with b instead of c :b)</p>

Reply:
<p>I was going by y=mx+c, but perhaps β0 and β1 would be petter?</p>

Reply:
<p>Yeah probably, that's less arcane -- or just <code>intercept + slope * x</code> , that works too here</p>

Reply:
<p>done</p>